src/main/java/com/pgssoft/httpclient/HttpClientMock.java

```diff
@@ -359,13 +359,19 @@ public void debugOff() {
         publisher.close();
         try {
             return subscriber.getBody().toCompletableFuture().get();
-        } catch (InterruptedException | ExecutionException e) {
+        } catch (ExecutionException e) {
+            var cause = e.getCause();
+            if (cause instanceof RuntimeException) {
+                throw (RuntimeException) cause;
```

Comment:
HttpClient in Java 11 has a different behaviour.
As bellow test shows:
```
  @Test
  public void foo() {
    var httpClient = HttpClient.newHttpClient();
    HttpRequest request = HttpRequest.newBuilder().GET().uri(URI.create("http://www.google.com")).build();
    BodyHandler<String> handler = (ri)->{throw new IllegalStateException();};
    assertThrows(IOException.class, ()->httpClient.send(request, handler));
  }
````
In case of exception in BodyHandler HttpClient return IOException, HttpClientMock should behave in the same way.